src/screens/Analytics.tsx

```diff
@@ -28,44 +228,86 @@ export function Analytics() {
               // style={{ marginBottom: 20, marginTop: 20, background: '#a2fb1b' }}
             >
               <Tab label="bitrate/fps" style={{ marginRight: 12 }} />
-              <Tab label="modem one bandwidth" style={{ marginRight: 12 }} />
-              <Tab label="modem Two bandwidth" style={{ marginRight: 12 }} />
-              <Tab label="modem Three bandwidth" style={{ marginRight: 12 }} />
               <Tab label="Aggregate upstream" style={{ marginRight: 12 }} />
               <Tab label="Aggregate downstream" style={{ marginRight: 12 }} />
+              <Tab label="Individual modems" style={{ marginRight: 12 }} />
             </Tabs>
-            {value === 0 && <ComposedTwoYAxisChart />}
-            {value === 1 && <ComposedTwoAreasChart />}
-            {value === 2 && <ComposedTwoAreasChart />}
-            {value === 3 && <ComposedTwoAreasChart />}
-            {value === 4 && <StackAreasChart />}
-            {value === 5 && <StackAreasChart />}
+            {value === 0 && <ComposedTwoYAxisChart data={streamData} />}
```

Comment:
a suggestion. Should we write a comment or change the value to match the tab name, make it readable for others?